pages/thanks.md

```diff
@@ -22,3 +21,17 @@ Those companies help us to develop JHipster
 *   EJ Technologies, which gives us free licences for the [JProfiler Java profiler](http://www.ej-technologies.com/products/jprofiler/overview.html)
 
 And a very special thanks to the development team from [Galeries Lafayette](http://www.galerieslafayette.com/), which had the original idea of making Grunt work with Ant, and create a modern Web front-end on top of a more classical Spring MVC application.
+
+# <i class="fa fa-gift"></i> Companies and projects contributing to JHipster
+
+JHipster has got a lot of [contributors](https://github.com/jhipster/generator-jhipster/graphs/contributors), and there are official contributions from popular projects and companies as well. Below is the list of companies and projects contributing to JHipster and we are very thankful for their contributions.
+
+*   [Ippon Technologies](http://www.ippon.fr/), which provides time and hardware to [@juliendubois](https://twitter.com/juliendubois) so he can improve the project
+*   [XebiaLabs](https://xebialabs.com/), which provides time and hardware to [@deepu105](https://twitter.com/deepu105) so he can improve the project
+*   [Google](https://google.com) contributed the Kubernetes(http://kubernetes.io/) sub generator. It is maintained by [Ray Tsang](https://twitter.com/saturnism)
+*   [Heroku](https://www.heroku.com/) contributed the Heroku sub generator. It is maintained by [Joe Kutner](https://twitter.com/codefinger)
```

Comment:
Any chance of getting some Stormpath love here? ;)

* [Stormpath](https://stormpath.com/) contributed the Stormpath sub generator. It is maintained by [Matt Raible](https://twitter.com/mraible)